### DISCUSSION TITLE
When connected to a monitor, allow the phone to turn off display

### DISCUSSION THREAD

**Discussion by cmlosariaGIS** [Archfx/duo-de#43] [Ideas]
Hello,

Thanks for putting this up. I plan to flash this. I use a Duo 1.

My suggestion would be, (if possible to implement) is when I connect a monitor to the duo, to allow the user to turn off the display but video out still works.

Cheers,

Charlie

**Comment by Archfx** [comment]
I know the Duo-de works with external monitors without any issues (yes it launches it in desktop mode), but I haven't tried your specific configuration with the device display turned off, however, it should work.
